examples/heliparasite.nwk

```diff
@@ -0,0 +1,3 @@
+((((emma_EastPE,(favorinus_EastPE,etylus_EastE)n5)n4,lativitta_EastE)n3,(erato_EastFG,hydara_EastFG)n6)n2,((((hydara_EastT,hydara_WestPA)n10,petiverana_WestCR)n9,petiverana_WestPA)n8,(hydara_EastC,cyrbia_WestE)n11)n7)n1;
```

Comment:
Same as others, renamed to `heliconius_parasite.nwk`.

Reply:
Removed.